### PR TITLE
[MRG] more stuff

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

File: 01-import_and_filter.py [hunk 1]
```diff
@@ -78,18 +78,17 @@ def run_filter(subject):
             raw.resample(config.resample_sfreq, npad='auto')
 
         raw.save(raw_fname_out, overwrite=True)
-        raws.append(raw)
```
Comment (agramfort) [inline 01-import_and_filter.py:81]: @SophieHerbst this avoids keeping all the raw files in memory just to know how many you have seen

File: 03-extract_events.py [hunk 2]
```diff
@@ -36,10 +36,10 @@ def run_events(subject):
                                  min_duration=config.min_event_duration,
                                  shortest_event=config.shortest_event)
 
-        if config.trigger_offset:
+        if config.trigger_time_shift:
```
Comment (agramfort) [inline 03-extract_events.py:39]: @SophieHerbst this is a rename to be more explicit.

File: 05a-run_ica.py [hunk 3]
```diff
@@ -67,6 +66,8 @@ def run_ica(subject, tsss=config.mf_st_duration):
                                 baseline=config.baseline,
                                 preload=True, decim=config.decim,
                                 reject=config.reject)
+    # XXX : config.reject should not be the same for ICA as here
+    # you want to keep the EOG blinks in
```
Comment (agramfort) [inline 05a-run_ica.py]: @SophieHerbst this we need to discuss

File: 01-import_and_filter.py (hunk 1 above)
Comment (SophieHerbst) [reply 01-import_and_filter.py:81]: great!

File: 03-extract_events.py (hunk 2 above)
Comment (SophieHerbst) [reply 03-extract_events.py:39]: +1

File: 05a-run_ica.py (hunk 3 above)
Comment (SophieHerbst) [reply 05a-run_ica.py]: @agramfort the eog is removed from the reject dict in config right now, but you are right, we should make it more explicit.

**Comment by agramfort** [comment]
Ok Green. Please @SophieHerbst if happy

**Pull request merged**